spec/support/content_store_helpers.rb

```diff
@@ -0,0 +1,45 @@
+module ContentStoreHelpers
+  RSpec::Matchers.define :have_content_item_slug do |expected_slug|
+    match do |content_store|
+      content_store.stored_slugs.include?(expected_slug)
+    end
+  end
+
+  RSpec::Matchers.define :have_draft_content_item_slug do |expected_slug|
+    match do |content_store|
+      content_store.stored_draft_slugs.include?(expected_slug)
+    end
+  end
+
+  def stub_content_store!
+    @content_store ||= FakeContentStore.new
+    allow(CollectionsPublisher).to receive(:services)
+      .with(:publishing_api)
+      .and_return(@content_store)
+
+    @content_store
+  end
+
+  alias :content_store :stub_content_store!
```

Comment:
Would `stubbed_content_store` be a better name for this alias to make it clear in the tests what's being referred to.

Also, the way this alias is used, it does the `allow(CollectionsPublisher)...` every time it's accessed.  Would be good to avoid that if possible.  Maybe make this a simple accessor instead of an alias.